draft-poidt-ccamp-actn-poi-pluggable.md

```diff
@@ -237,29 +264,184 @@ Px = DWDM (coherent pluggable) Router ports
+
+   Two different architectural options have been identified, namely:
+~~~~ ascii-art
+     - Option 1: Dual SBI management of IPoDWDM routers
+     - Option 2: Single SBI management of IPoDWDM routers
+~~~~
+
+   Both the options foresee that the packet and the optical domain are 
+   kept separate from a management perspective.
+
+
+## Option 1 - Dual SBI management of IPoWDM routers
+
+The figure {{my_figure-2}} describes the architecture of option 1.
+
+
+~~~~ ascii-art
+
+                        +----------+
+                        |   MDSC   |
+                        +--+----+--+
+                           |    |  
+  MPI interface    +-------+    +-------+
+                   |                    | 
+              +----+----+          +----+----+
+              |  P-PNC  |          |  O-PNC  |
+              +--+---+--+          +----+----+
+                 ^   ^               ^  ^  ^
+                 |   |               |  |  |
+         +-------+   +---------------|--|--|------+
+         |                           |  |  |      |  
+         |   +-----------------------+  |  +--+   |
+         |   |               +----------+     |   |
+         |   |               |                |   |
+         |   |               v                |   |
+         v   |        +--------------+        |   v
+        +-----+      /                \      +-----+
+       / P.N.1 \..../..O.N--------O.N..\..../ P.N.2 \
+       \       /    \  Optical Domain  /    \       / 
+        +-----+      \                /      +-----+
+                      +--------------+
+
+   P.N. = Packet/Optical Node (IPoWDM router) 
+   O.N. = Optical Switching DWDM Node (ROADM)
+   ROADM = Lambda/Spectrum switch
+~~~~
+{: #my_figure-2 title="Dual SBI management Scenario"}
+
+
+   The peculiarity of this option consists of the fact that both the 
+   packet SDN controller (P-PNC) and the optical SDN controller (O-PNC)
+   have access to the coherent pluggable optics on the routers. 
+   The P-PNC is the only entity allowed to configure them, while the
+   O-PNC is granted with read-only permissions to avoid database 
+   inconsistency between them. Data write access permissions are 
+   expected to be implemented on the routers to only grant 
+   configuration rights to the P-PNC.
+
+   The read-only capabilities of the O-PNC consist in the following 
+   tasks:
+     - Device discovery, poll or stream configuration, state and 
+       static capabilities.
+     - Performance monitoring, periodically poll or stream performance 
+       counters.
+     - Fault notification, received asynchronous alarm notifications.
+
+   The consequence of this split of roles is that the P-PNC exposes
+   the coherent pluggables configuration APIs, while the O-PNC only
+   the APIs needed for path computation (for OTSi services planning)
+   and service provisioning for OLS media channel services.
+
+
+ ## Option 2 - Single SBI management of IPoWDM routers
```

Comment:
I propose, that minimally an option 3 be added that recognizes the domain boundary in the device and explains that the optical controller controls the photonics of the plug. I would be happy to write up this option (in a consistent form with the other options) and provide the use case details.